test/commands/hub-group-api.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from '@oclif/test'
```

Comment:
move this test to the proper folder (test/api)

Reply:
done